maskit/examples/load_data.py

```diff
@@ -0,0 +1,14 @@
+from maskit.examples.load_circles import load_circles
+from maskit.examples.load_iris import load_iris
+from maskit.examples.load_mnist import load_mnist
+
+
+def load_data(dataset, wires=None, embedding=None, params=None):
```

Comment:
Is `embedding` used at any point? Or for future features?

Reply:
It was intended for some future features, but using `**kwargs` as suggested above it shouldn't be a problem in the future.